crates/types/src/robot_dimensions.rs

```diff
@@ -3,6 +3,7 @@ use nalgebra::{vector, Vector3};
 #[derive(Debug)]
 pub struct RobotDimensions {}
 
+// TODO: consider wrapping in coordinate systems
```

Comment:
We definitely should do that!

Reply:
Agreed, but last time I tried this I ran into const issues.

Reply:
Done